model/activist.go

```diff
@@ -1131,12 +1146,14 @@ func UpdateActivistData(db *sqlx.DB, activist ActivistExtra, userEmail string) (
 	// TODO: make this work for other chapters too, maybe based on Chapters table mailing list info?
 	// or maybe just passing chapter Name or ID to the signup service?
 	if origActivist.ChapterID == 47 {
-		activistInfoChanged := activist.Name != origActivist.Name ||
+		activistAddressChanged := activist.StreetAddress != origActivist.StreetAddress ||
+			activist.City != origActivist.City ||
+			activist.State != origActivist.State
+		activistInfoChanged := activistAddressChanged ||
+			activist.Location != origActivist.Location ||
```

Comment:
should location be considered in activistAddressChanged instead? it look like it contains the Zip which is used by the mailing list.

but then Zip is not used by the geo API, so maybe this logic should not be shared between geo api logic and mailing list logic, in which case using names like "isMailingListInfoChanged" and "isGeoInfoChanged" would better convey the intent.

Reply:
Yeah I don't think zip is used for geocoding so I'll do the latter